tap_stripe/__init__.py

```diff
@@ -428,6 +437,16 @@ def get_bookmark_for_stream(stream_name, replication_key):
             int(utils.strptime_to_utc(Context.config["start_date"]).timestamp())
     return stream_bookmark
 
+def get_bookmark_for_sub_stream(stream_name):
```

Comment:
Add comments on each line, how we are collecting bookmarks for the child stream!

Reply:
Added comments

Reply:
Also add function comment for each function